backend/src/lib/requestPromisified/index.js

```diff
@@ -0,0 +1,24 @@
+import request from 'request';
+
+// this function create the normal request object in a promise.
+const requestPromisified = ({
```

Comment:
this is a duplicate code from file ` backend/src/lib/requestPromisified.js`

Reply:
I will remove it and just keep the folder instead.